test/semantics/CMakeLists.txt

```diff
@@ -91,6 +91,8 @@ set(ERROR_TESTS
   computed-goto02.f90
   nullify01.f90
   nullify02.f90
+  deallocate01.f90
+  deallocate04.f90
```

Comment:
Is there `deallocate02.f90` and `deallocate03.f90` somewhere?

Reply:
Yes.  Tests 02 and 03 are still in my sandbox.  They have errors that are caught by the parser.  I want to talk to Peter about them before I commit them.